host/src/README.md

```diff
@@ -1,11 +1,7 @@
+# FunctionsNetHost
 
-This is the project root where we have our root level CMakeLists.txt.
-### Load & Build. 
+Managed code version of FunctionsNetHost.
```

Comment:
After this is merged, without more context about the fact that we once had a native implementation, this may become confusing. Simply calling this the `Azure Functions Worker .NET Host` would be better.

Reply:
Sure. I pushed a newer version with a bit more text on the README. This is something I would love to get more help on.